jetstream/core/orchestrator.py

```diff
@@ -502,11 +502,6 @@ def _prefill_thread(self, idx: int):
       padded_tokens, true_length = self._process_prefill_content(
           request, tokenizer, is_bos, prefill_engine.max_prefill_length
       )
-      if isinstance(prefill_engine, engine_api.JetStreamEngine):
-        request.padded_token_length = token_utils.take_nearest_length(
```

Comment:
can we remove padded_token_length: https://github.com/google/JetStream/pull/92/files#diff-753df909960695fd6aa61b95c4762c4edc50a298007d40de7096a7e49e0e1215R138, and the extra new line: https://github.com/google/JetStream/pull/92/files#diff-753df909960695fd6aa61b95c4762c4edc50a298007d40de7096a7e49e0e1215R519